### PR TITLE
chore: spaces plz

### PR DESCRIPTION
**Pull request by PatrickJS**

### CODE REVIEW

File: src/build.js [hunk 1]
```diff
@@ -5,69 +5,104 @@ const path = require('path');
...
-		: 'other';
+  const scope = SCOPES.includes(getFirstChar(name))
+    ? NUMBERS_MAP[getFirstChar(name)]
+      ? NUMBERS_MAP[getFirstChar(name)]
+      : getFirstChar(name)
+    : 'other';
 
-	if (!(scope in packages)) packages[scope] = [];
-	packages[scope].push(name);
+  if (!(scope in packages)) {
```
Comment (uncenter) [inline src/build.js:40]: why it was better before smh

Comment (PatrickJS) [reply src/build.js:40]: haha my vscode and ai tools were flipping out and auto messing up things

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+# Indentation override for all JS under lib directory
+[lib/**.js]
+indent_style = space
+indent_size = 2
+
+# Matches the exact files either package.json or .travis.yml
+[{package.json,.travis.yml}]
+indent_style = space
+indent_size = 2
```
Comment (uncenter) [inline .editorconfig:33]:
```suggestion
root = true

[*]
indent_style = space
indent_size = 2
end_of_line = lf
insert_final_newline = true
trim_trailing_whitespace = true
charset = utf-8
```

File: src/build.js (hunk 1 above)
Comment (uncenter) [reply src/build.js:40]: wdym

Comment (PatrickJS) [reply src/build.js:40]: I have too many vscode extensions that keep auto suggesting and refactoring the code weird 😂 definitely a me problem

**Pull request merged**